pkg/agent/openflow/multicast.go

```diff
@@ -112,6 +118,18 @@ func (f *featureMulticast) multicastReceiversGroup(groupID binding.GroupIDType,
 	return group
 }
 
+// multicastForwardInboundGroup forwards ingress multicast traffic sent from FlexibleIPAM Pods to host interface
+// and multicast pipeline.
+func (f *featureMulticast) multicastForwardInboundGroup(groupID binding.GroupIDType, tableID uint8) binding.Group {
+	group := f.bridge.NewGroupTypeAll(groupID).ResetBuckets()
```

Comment:
No need to call `ResetBuckets` in a new group as we do not save group in ofnet.groupDB any more.

Reply:
updated